modules/kafka/kafka-dest-driver.h

```diff
@@ -35,27 +35,35 @@ typedef struct
   LogTemplateOptions template_options;
   LogTemplate *key;
   LogTemplate *message;
+  LogTemplate *topic_name;
+  GHashTable *topic_hash_map;
+  GMutex lock;
 
-  gchar *topic_name;
   GList *config;
   gchar *bootstrap_servers;
+  gchar *fallback_topic_name;
   rd_kafka_topic_t *topic;
```

Comment:
is "topic" still needed?

Reply:
I was using it for the case where the topic name isn't a template so that I can initialise it in the driver. Is there a way to get rid of it without increasing complexity?

Reply:
ok, understood.